update derpitcraft (#2)
TrixNetwork/docs: Derpitcraft/README.md

# Derpitcraft Season 9: Rules and Guidelines
## A brief introduction and some important details
Derpitcraft is a **vanilla-like** economy SMP, for both Java and Bedrock, with cracked-support. It has a **friendly** community and best-in-class **anti-grief** measures. There is a **whitelist** to ensure the best multiplayer minecraft experience!

> You can play Derpitcraft by registering for it: Use `/dcregister` in any channel.

Reviewer: ```suggestion
You can play Derpitcraft by registering for it: Use `/dcregister` in any discord channel.
```

> You can play Derpitcraft by registering for it: Use `/dcregister` in any discord channel.

Reviewer 2: ```suggestion

You can play Derpitcraft by registering for it: Use `/dcregister` in any discord channel.
```

keep this in seperate paragraph for easier understanding.

### Important details
#### Version support

> Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.x**.

Reviewer: ```suggestion
Minecraft **Bedrock Edition** `1.18.0 - 1.18.10` and Minecraft **Java Edition** `1.18.x`.
```

#### How to join

> Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **Derpitcraft Season 9** NPC to join. You can join the SMP after **April 20th 2022**.

Reviewer: ```suggestion
Run `$ip` on our discord to find the IPs for both bedrock and java. Join **TrixMC** using the IP. **Right-click**  the **Derpitcraft Season 9** NPC to join (If you are on mobile, you can tap and hold the NPC.). You can join the SMP after **April 20th 2022**.
```

#### Spawn coordinates
**Coordinates [X, Z]:**: `[-161, -223]` 

> This is world center. There is a world border at **7500 blocks**.

Reviewer: ```suggestion
This is the world centre. There is a world border at **7500 blocks** on each axis.
```

> This is the world centre. There is a world border at **7500 blocks** on each axis.

Reviewer 2: ```suggestion

This is the world centre. There is a world border at **7500 blocks** on each axis.
```
keep this in sep. para for easier understanding

#### Shopping District coordinates
**Coordinates [X, Z]:**: `[-250, -262]`

> Scroll down to read more about what the Shopping District is (**Point 5**).

Reviewer 2: ```suggestion
Scroll down to read more about what the Shopping District in (**Point 5**).
```

## Rules and Guidelines
### Rule Of Thumb
All the following rules can be summarised simply, in one line. "Be kind to others on the server". This server follows a Honor Code, and we expect everyone to respect each other. Disrespect is actionable upon. We want to ensure a safe and friendly experience for everyone, and this is not possible without your co-operation. We kindly request you to follow all these rules and keep everything playful and fun.
### Detailed Rules and Guidelines
1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Point 2**). It includes *harmful* pranking. Read more about the pranking policy in **Point 3**. For a better idea on what consists of a player's base, structure, or farm, see **Point 4**.

>   > To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.

Reviewer: ```suggestion
To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
```

> To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.

Reviewer 2: Markdown won't render this separately, add new line between paragraphs, also indent in to make markdown understand its part of the point.

```suggestion

    To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.
```

Reviewer 2: if it renders correctly please attach a screenshot im not sure it will

2.  **PvP Policy**: PvP is not permitted on the server except under specific conditions. PvP includes killing a player's pet, and damaging the player in any way, including but not limited to: using TNT, using weapons, using bows, using crossbows, using potions, and using pets. 
    - *PvP is permitted for self-defense*: If someone else attacks first, you can PvP to defend yourself. Don't escalate a fight for self-defense by using a better weapon. This is likely to make things worse. If it is a fist fight, try to resolve it peacefully without weapons at all. We do not recommend PvP to defend your base, structures, or farms. If someone is actively griefing your base, report the player *immediately* (see **Reporting**), and try to make the other party stop using the *chat*.
   
    - *PvP is permitted with consent*: If two or more players agree to PvP, they can PvP based on their own internal terms. We will not intervene unless a player is reported or griefing is detected. If both parties agree, you can have playful fights, kill the other party, and even make arena's for the entire server to enjoy.
   

>     > PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.

Reviewer: ```suggestion
PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.
```

> PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.

Reviewer 2: Markdown won't render this separately, add new line between paragraphs, also indent in to make markdown understand its part of the point.
```suggestion
    PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.
```

3. **Pranking Policy**: Harmful pranks are not allowed. Harmless pranks are allowed and encouraged to create a fun experience for all players. Harmful pranking includes pranks that cause deaths, damage to items, and loss of items. Don't play pranks which won't be taken lightly. Try to judge this based on the player you are pranking. Try to take all kinds of harmless pranks in a light and fun manner. If you prank someone in a way that makes them die, or lose items, you are responsible for the items and need to return them within an hour. If you can't return them, then it will be counted as griefing. Always try to use common sense, and don't do pranks that can't be reversed. Think before pranking. If you don't want to be pranked at all, you can explicitly tell players to not prank you. If they don't respect your decision, report them, and we will take action. 

> 4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the `/dccoordinates` command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check [`#ds9-coordinates`](https://discord.com/channels/297301054930944011/965230296226222150) to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare / lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Point 5**). 

Reviewer: ```suggestion
4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the `/dccoordinates` command on our discord to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check [`#ds9-coordinates`](https://discord.com/channels/297301054930944011/965230296226222150) to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare/lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Point 5**). 
```

Author: looks cleaner with space

> 5. **Shopping District Policy**: Like Hermitcraft, Derpitcraft has a *Shopping District* (SD). It is the centrally located Mooshroom Island located at the coordinates given above. It is a place where all players can create community *shops* to trade items for services or other items You **cannot** break blocks or build on the shopping district, **unless** you are building a shop (or you are stuck!). You have to **pay** to claim land in the SD. The SD is a place where griefing is strictly denied. There will be heavy consequences to rule breaking in the SD.

Reviewer: ```suggestion
5. **Shopping District Policy**: Like [Hermitcraft](https://hermitcraft.fandom.com/wiki/Hermitcraft), Derpitcraft has a *Shopping District* (SD). It is the centrally located Mooshroom Island located at the coordinates given above. It is a place where all players can create community *shops* to trade items for services or other items You **cannot** break blocks or build on the shopping district, **unless** you are building a shop (or you are stuck!). You have to **pay** to claim land in the SD. The SD is a place where griefing is strictly denied. There will be heavy consequences to rule breaking in the SD.
```

>    - *Plots in the SD*: You can buy "plots" of land in the SD. These plots are shaped as any simple geometric shape or a combination of them. The suggested shape for a shop is a **square** or **rectangle**. Making multiple shops on the same plot is frowned upon, but not strictly denied. You can buy multiple plots and merge them together for larger shops.

Reviewer: ```suggestion
   - *Plots in the SD*: You can buy `plots` of land in the SD. These plots are shaped as any simple geometric shape or a combination of them. The suggested shape for a shop is a **square** or **rectangle**. Making multiple shops on the same plot is frowned upon, but not strictly denied. You can buy multiple plots and merge them together for larger shops.
```

Author: not needed tbh

Reviewer 2: i agree, do not do this they are "plots" because you don't really call things "plots" in minecraft. It's a given name, not something to be highlighted.

   - *Paying for plots*: Payment is done by placing **one diamond block** on the pedestal in the shopping district. **One diamond block** buys you a plot of **144 blocks** in size. This payment is **not refundable**. You can start building on the SD after you have paid. Once you pay, you can mark your shop's area using a border.
   
       **WARNING**: You **CANNOT** break blocks once you place them on the pedestal, so **DONT PLACE ANY OTHER BLOCK**. Raise a ticket if you make a mistake.

>    - *Other details*: Plots **can** be asked to be by moderators if deemed necessary. They **cannot** be within 2 blocks of each other. Respect other's plots. Don't fill the whole district with shops! Instead, build roads and pathways!

Reviewer: ```suggestion
   - *Other details*: Plots **can** be asked to be removed by moderators if deemed necessary. They **cannot** be within 2 blocks of each other. Respect other's plots. Don't fill the whole district with shops! Instead, build roads and pathways!
```

6. **Chat Policy**: *Spamming* in chat is **not** allowed. *Excessive swearing* is **not** allowed. Do **not** *insult* other players. Severe action can be taken if you are reported. Our server has *team chats*. These are relatively unmoderated. Please read the guide on **Teams** for more information. This extends to voice chat's using the Proximity Chat system on Derpitcraft.
   
7. **Spectator Account Policy**: Spectator accounts will not be allowed this season, except for special cases. If you need a spectator account for any reason, please raise a ticket. Players can use the Replay mod (https://www.replaymod.com/) to create time-lapse's, etc. as an alternative. It is recommended to use this mod.
   

> 8.  **Resource Pack Policy**: **Cheating** respource packs are not allowed. This includes, but is not limited to, **X-RAY** resource packs.You can only use normal resource packs. If we detect you as using a resource pack that gives you an unfair advantage, we can take action.  Let us know if you are using a resource pack in [`#ds9-general`](https://discord.com/channels/297301054930944011/965132663910436914).

Reviewer: ```suggestion
8.  **Resource Pack Policy**: **Cheating** respource packs are not allowed. This includes, but is not limited to, **X-RAY** resource packs.You can only use resource packs that do not give you an unfair advantage. If we detect you as using a resource pack that gives you an unfair advantage, we will take action.  It is a good idea to let us know if you are using a resource pack in [`#ds9-general`](https://discord.com/channels/297301054930944011/965132663910436914).
```

9.  **Clients, Mods, and More Policy**: The following lists *allowed* modifications to the game. Anything else should be considered not allowed. Raise a ticket if you are unsure.
    1.  Badlion, Lunar, Salwyrr, TLauncher, MultiMC and the official Minecraft Launcher are allowed. **Ask permission in a ticket for other clients**.
    2. Xaero's Minimap and Xaero's Worldmap with **fair play mode on**.
    3. miniHUD / CustomHUD Mod
    4. Replay Mod 
    5. Optifine
    6. Multiple fabric performance enhancing mods are allowed. A comprehensive list of these cannot be made. If you think a mod will not be allowed, *raise a ticket* before using it.
    **Important**: Gamma and full-bright are **not allowed**.

>     Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal verison. The lite version is considerably smaller.

Reviewer: ```suggestion
    Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal version. The lite version is considerably smaller and designed for lower-end computers.
```

>     **Important**: Gamma and full-bright are **not allowed**.

Reviewer 2: It needs to be a new paragraph!!!

```suggestion

    **Important**: Gamma and full-bright are **not allowed**.
```

>     **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. Download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. To read more on how to use this system, read the guide on it below.

Reviewer: ```suggestion
    **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the [`Simple Voice Chat`](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat) mod. Download this mod from [here](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat) to play with Proximity Chat on Derpitcraft. **It is optional**. To read more on how to use this system, read the guide on it below.
```

> 10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. **Raise a ticket if this is absolutely necessary.**

Reviewer: ```suggestion
10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are an **offline mode** server, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. **Raise a ticket if this is absolutely necessary.**
```

Author: cracked is more clearer, people might not know what offline mode is

Reviewer 2: yeah no one knows

>     Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal version. The lite version is considerably smaller and designed for lower-end computers.

Reviewer 2: ```suggestion
    Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal version. The lite version is considerably smaller and designed for lower-end computers.
    
```

the next line needs to be in a new paragraph!

  
     - Remember, lag is both **server-side** and **client-side**. We can only compensate for **server-side** lag, and that also, to an extent. It is recommended to keep your X, Y, and Z coordinates on the screen at all times by either using a client-side Mod, or using the *coordinates HUD datapack* installed on the server (Read the how to guide on **commands** for more information).
  
     - This is a **server**, and all items will **despawn** after **5 minutes**, regardless of if you are online or not. The items may even despawn in chunks that you think are unloaded, but get loaded by somebody else. **It is recommended to assume that all items despawn after 5 minutes**. We cannot provide you your items back if they have despawned.
This list is ***not comprehensive***. If you think something may not be allowed, ask us by raising a ticket. If you have any other questions, please raise a ticket. The use of loopholes in these rules is not permitted, and will result in punishment. The say of the moderators is final in all cases. 
## Compatibility Issues on Bedrock

> There might be a few compatibility issues with the server, since we support both Java and Bedrock. None of these issues pre gameplay. You may need to ask a Java edition player to *create maps*, etc. If you are experiencing issues with the server, please mention a moderator.

Reviewer: ```suggestion
There might be a few compatibility issues with the server, since we support both Java and Bedrock. None of these issues breaks gameplay. You may need to ask a Java edition player to *create maps*, etc. If you are experiencing issues with the server, please mention a moderator.
```

Author: breaks shud be break

Reviewer 2: ```suggestion
There might be a few compatibility issues with the server, since we support both Java and Bedrock. None of these issues break gameplay. You may need to ask a Java edition player to *create maps*, etc. If you are experiencing issues with the server, please mention a moderator.
```

> This list is ***not comprehensive***. If you think something may not be allowed, ask us by raising a ticket. If you have any other questions, please raise a ticket. The use of loopholes in these rules is not permitted, and will result in punishment. The say of the moderators is final in all cases. 

Reviewer 2: this needs to DEFINITELY be in a new para. Add a line here
```suggestion

This list is ***not comprehensive***. If you think something may not be allowed, ask us by raising a ticket. If you have any other questions, please raise a ticket. The use of loopholes in these rules is not permitted, and will result in punishment. The say of the moderators is final in all cases. 
```

### Important details

#### Version support
Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.2**.
Minecraft **Bedrock Edition 1.18.0 - 1.18.10** and Minecraft **Java Edition 1.18.x**.

#### How to join
Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **DERPITCRAFT SEASON 9** NPC to join. You can join the SMP after **April 20th 2022**.
Run `$ip` to find the IPs for both bedrock and java. Join the **TrixMC Network** using the IPs. **Right click** on the **Derpitcraft Season 9** NPC to join. You can join the SMP after **April 20th 2022**.

#### Spawn coordinates

**spawn coordinates**: `[-161, -223]` 
**Coordinates [X, Z]:**: `[-161, -223]` 

This is world center. There is a world border at **7500 blocks**.

#### Shopping District coordinates

**shopping district center coordinates**: `[-250, -262]`
**Coordinates [X, Z]:**: `[-250, -262]`

> Scroll down to read more about what the Shopping District is (**Pt.5**).

Reviewer 2: They are points. If you want to make them parts, then make them all headings and make these links.

### Detailed Rules and Guidelines

> 1. **No Griefing Policy**: Griefing is not tolerated at all. If you are caught, you will be **banned** from Derpitcraft Season 9 indefinitely. Our moderators will decide a suitable punishment. Griefing also includes stealing items from other players, destroying their bases, structures, and farms, and causing deaths / killing players (The PvP policy is explained in **Part 2**). It includes *harmful* pranking. Read more about the pranking policy in **Part 3**. For a better idea on what consists of a player's base, structure, or farm, see **Part 4**.

Reviewer 2: They are points. If you want to make them parts, then make them all headings and make these links.

>   > To report griefing, or a player in any other condition, read the **Reporting** section of the **How To Guide**.

Reviewer 2: It's not supposed to be a quotation block. How is this rendered?

Author: it looks cleaner

2.  **PvP Policy**: PvP is not permitted on the server except under specific conditions. PvP includes killing a player's pet, and damaging the player in any way, including but not limited to: using TNT, using weapons, using bows, using crossbows, using potions, and using pets. 

    - *PvP is permitted for self-defense*: If someone else attacks first, you can PvP to defend yourself. Don't escalate a fight for self-defense by using a better weapon. This is likely to make things worse. If it is a fist fight, try to resolve it peacefully without weapons at all. We do not recommend PvP to defend your base, structures, or farms. If someone is actively griefing your base, report the player *immediately* (see **Reporting**), and try to make the other party stop using the *chat*.
   
    - *PvP is permitted with consent*: If two or more players agree to PvP, they can PvP based on their own internal terms. We will not intervene unless a player is reported or griefing is detected. If both parties agree, you can have playful fights, kill the other party, and even make arena's for the entire server to enjoy.
   

>     - PvP fights and discrepancies are judged on a case to case basis, and an investigation will take place if a report is raised. This policy extends to harming players in any way.

Reviewer 2: It's not a point. The points are specifically for PvP being not maintained in these conditions. This is a general policy extension.

Author: made it a quotation, to signify it as a general policy extension(its not the actual use case but it makes it look like that

3. **Pranking Policy**: Harmful pranks are not allowed. Harmless pranks are allowed and encouraged to create a fun experience for all players. Harmful pranking includes pranks that cause deaths, damage to items, and loss of items. Don't play pranks which won't be taken lightly. Try to judge this based on the player you are pranking. Try to take all kinds of harmless pranks in a light and fun manner. If you prank someone in a way that makes them die, or lose items, you are responsible for the items and need to return them within an hour. If you can't return them, then it will be counted as griefing. Always try to use common sense, and don't do pranks that can't be reversed. Think before pranking. If you don't want to be pranked at all, you can explicitly tell players to not prank you. If they don't respect your decision, report them, and we will take action. 

> 4. **Land Claim Policy**: Claiming land is done through mutual respect. Do not make bases near other players/ teams. Give each player respectable space, and leave space for them to expand. You can lay "claims" to land for farms, structures, and land for bases. You can claim land by writing your name on a sign, and placing it in an area. You can also mark an area with a border. Further, you can use the `/dccoordinates` command to announce your claim on Discord (Read the **Coordinates and Claims** how to guide). It is recommended to check `#ds9-coordinates` to see if an area is not already claimed. You can't claim strongholds. The stronghold, once found, is **community property** and anyone is free to use it. You can claim villages, though it is encouraged to spare / lend a few villagers for your fellow players so they can make their own breeders. Raise a report if someone has settled too close to you. This doesn't apply to claims on the Shopping District Island (See **Part 5**). 

Reviewer 2: They are points. If you want to make them parts, then make them all headings and make these links. 

Link the channel if you can.

Author: will link

   
7. **Spectator Account Policy**: Spectator accounts will not be allowed this season, except for special cases. If you need a spectator account for any reason, please raise a ticket. Players can use the Replay mod (https://www.replaymod.com/) to create time-lapse's, etc. as an alternative. It is recommended to use this mod.
   

> 8.  **Resource Pack Policy**: **Cheating** respource packs are not allowed. This includes, but is not limited to, **X-RAY** resource packs.You can only use normal resource packs. If we detect you as using a resource pack that gives you an unfair advantage, we can take action.  Let us know if you are using a resource pack in `#ds9-general`.

Reviewer 2: Link the channel if you can.

Author: will link

        3. You can craft droppers to dispensers. 
        4. You can dye sandstone and universally dye blocks. 
        5. You can make shulkers in any shape. 
        6. Blackstone can be used everywhere cobblestone is used in crafting recipies. 
        7. You get more trapdoors, bark, stairs and bricks upon crafting. 
        8. You can unpack ice, nether wart blocks and wool.
    12. Several other custom commands have been added. These are all listed in the How To Guides
## How to guides
### Reporting Players

> As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing `TrixMC` issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.

Reviewer 2: quotes make a lot more sense here. Its a given name to the issue type. not really a command or something
```suggestion
As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing "TrixMC" issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.
```

### Teams and the AFK System
To declare a team, raise a ticket. In the ticket, you will be asked to mention who the team members are, and the preffered team colour and Minecraft Prefix that will show up in-game. Teams have slightly altered rules for slime chunks (see the next how to guide). A team can have a maximum of **8 members**.

> To message within your team, use the `/tmsg <msg>` command in-game.

Reviewer 2: ```suggestion

To message within your team, use the `/tmsg <msg>` command in-game.
```
seperate paragraph since we are talking about a whole different topic (Creating a team vs Messaging within a team)

#### AFK System
If you are stationary for more than a set amount of time, you will automatically be put into the AFK team. You can't use `/tmsg` to chat while you are in the AFK team. To get out of AFK, just move around. 

> You can also set yourself as AFK manually by using `/afk`. This will mark you as AFK for the benefit of everyone on the server.

Reviewer 2: Separate paragraph to introduce the new topic of the command.
```suggestion

You can also set yourself as AFK manually by using `/afk`. This will mark you as AFK for the benefit of everyone on the server.
```

### Getting Slime Chunks
Slime chunks are a fairly important part of the game. Raise a ticket to get a slime chunk close to your coordinates.

#### Slime Chunk Limits

> You can't get an infinte number of slime chunks. The limits imposed are as follows:

Reviewer 2: ```suggestion
You can't get an infinite number of slime chunks. The limits imposed are as follows:
```

> You can't get an infinite number of slime chunks. The limits imposed are as follows:

Reviewer 2: ```suggestion

```

    Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal verison. The lite version is considerably smaller.

    **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. Download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. To read more on how to use this system, read the guide on it below.

10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. **Raise a ticket if this is absolutely necessary.**

11.  **Lag Policy**: If you are experiencing lag, please wait a while for it to pass. If it stays and worsens, ping a moderator after **3 minutes** of continuous lag. Keep the following in your mind.

>      - If you think you have died due to a lag spike, TPS drop(under 5 TPS), or bad ping(more than 1000 ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.  

Reviewer 2: "
     - If you think you have died due to a lag spike, TPS drop (under 5 TPS), or bad ping (more than 1000ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.
"

Author: ?

Reviewer 2: edit it with the changed text. changed spacing

Reviewer 2: ```suggestion
    - If you think you have died due to a lag spike, TPS drop (under 5 TPS), or bad ping (more than 1000ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items. 
```

        5. You can make shulkers in any shape. 

>         6. Blackstone can be used everywhere cobblestone is used. 

Reviewer: ```suggestion
        6. Blackstone can be used everywhere cobblestone is used in crafting recipies. 
```

        7. You get more trapdoors, bark, stairs and bricks upon crafting. 
        8. You can unpack ice, nether wart blocks and wool.
    12. Several other custom commands have been added. These are all listed in the How To Guides
## How to guides
### Reporting Players

> As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing "TrixMC" issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.

Reviewer: ```suggestion
As a rule of thumb, try to resolve issues peacefully, without moderator intervention. If this doesn't work, report the player by raising a ticket (using `$rt`) and choosing `TrixMC` issue in the select menu. Please include the Player's IGN in your report if you can. It is important to take **screenshots** and even **video recordings** as evidence in some cases. If you feel like your issue is not adequeately solved by the moderators, ping **morpheus** directly.
```

### Teams and the AFK System
To declare a team, raise a ticket. In the ticket, you will be asked to mention who the team members are, and the preffered team colour and Minecraft Prefix that will show up in-game. Teams have slightly altered rules for slime chunks (see the next how to guide). A team can have a maximum of **8 members**.
To message within your team, use the `/tmsg <msg>` command in-game.
#### AFK System
If you are stationary for more than a set amount of time, you will automatically be put into the AFK team. You can't use `/tmsg` to chat while you are in the AFK team. To get out of AFK, just move around. 
You can also set yourself as AFK manually by using `/afk`. This will mark you as AFK for the benefit of everyone on the server.
### Getting Slime Chunks

> Slime chunks are a fairly important Point of the game. Raise a ticket to get a slime chunk close to your coordinates.

Reviewer: ```suggestion
Slime chunks are a fairly important part of the game. Raise a ticket to get a slime chunk close to your coordinates.
```

#### Slime Chunk Limits

> You can't get an infinte number of slime chunks. The limits imposed are as follows:

Reviewer: why twice?

Author: make it infinite
both spellings r wrong xd

Reviewer 2: > why twice?

what is twice?

Reviewer: Look next line

- `/trigger nc_inNether`: Converts current coordinates to overworld coordinates. Can be used as a way to efficiently calculate coordinates for portals.
- `/trigger nc_inOverworld`: Converts current coordinates to nether coordinates.Can be used as a way to efficiently calculate coordinates for portals.
- `/aboutdc`: Shows information about Derpitcraft.
- `/tm | /tmsg | /teammsg`: Message within your team.
### Using proximity chat (Simple Voice Chat)

> Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. You can download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.

Reviewer: ```suggestion
Derpitcraft has a proximity chat system powered by the [`Simple Voice Chat mod`](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat). You can download this mod from [here](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat) to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.
```

> Once you have downloaded the mod, it should work out of the box. Read more about the mod on it's wiki (https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.

Reviewer: ```suggestion
Once you have downloaded the mod, it should work out of the box. Read more about the mod on its [wiki](https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.
```

> You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard.

Reviewer: ```suggestion
You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard. Groups allow members to talk to each other even when they are not in close proximity.
```

### Coordinates Command and Claims
Derpitcraft has a rudimentary claims system. As explained earlier, claims are done based on mutual respect. To make it easier for you, we have a [`#ds9-coordinates`](https://discord.com/channels/297301054930944011/965230296226222150) channel. This channel is updated by our bot.
To update and add your coordinates to this channel, you can use the `/dccoordinates` command. After this, you can enter in the type of structure the coordinates are for, the X, Y, and Z coordinates, and a name for the structure. You can also choose to register coordinates for your TEAM as a whole, or for only you. Use the team feature if you want to register coordinates for your team!

> The coordinates get updated and posted into the dccoordinates channel. These coordinates also help us keep track of where someone's base is located. Always keep track of where your claims are located to make solving issues easier. These coordinates may be required in tickets.

Reviewer: ```suggestion
The coordinates get updated and posted into the [#ds9-coordinates](https://discord.com/channels/297301054930944011/965230296226222150) channel. These coordinates also help us keep track of where someone's base is located. Always keep track of where your claims are located to make solving issues easier. These coordinates may be required in tickets.
```

- `/trigger nc_inOverworld`: Converts current coordinates to nether coordinates.Can be used as a way to efficiently calculate coordinates for portals.
- `/aboutdc`: Shows information about Derpitcraft.

> - `/tm | /tmsg | /teammsg`: Message within your team.

Reviewer 2: ```suggestion
- `/tm` | `/tmsg` | `/teammsg`: Message within your team.
```

### Using proximity chat (Simple Voice Chat)
Derpitcraft has a proximity chat system powered by the [`Simple Voice Chat mod`](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat). You can download this mod from [here](https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat) to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.

> Once you have downloaded the mod, it should work out of the box. Read more about the mod on its [wiki](https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.

Reviewer 2: ```suggestion

Once you have downloaded the mod, it should work out of the box. Read more about the mod on its [wiki](https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.
```

> You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard. Groups allow members to talk to each other even when they are not in close proximity.

Reviewer 2: ```suggestion

You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard. Groups allow members to talk to each other even when they are not in close proximity.
```

### Coordinates Command and Claims
Derpitcraft has a rudimentary claims system. As explained earlier, claims are done based on mutual respect. To make it easier for you, we have a [`#ds9-coordinates`](https://discord.com/channels/297301054930944011/965230296226222150) channel. This channel is updated by our bot.

> To update and add your coordinates to this channel, you can use the `/dccoordinates` command. After this, you can enter in the type of structure the coordinates are for, the X, Y, and Z coordinates, and a name for the structure. You can also choose to register coordinates for your TEAM as a whole, or for only you. Use the team feature if you want to register coordinates for your team!

Reviewer 2: ```suggestion

To update and add your coordinates to this channel, you can use the `/dccoordinates` command. After this, you can enter in the type of structure the coordinates are for, the X, Y, and Z coordinates, and a name for the structure. You can also choose to register coordinates for your TEAM as a whole, or for only you. Use the team feature if you want to register coordinates for your team!
```

> The coordinates get updated and posted into the [#ds9-coordinates](https://discord.com/channels/297301054930944011/965230296226222150) channel. These coordinates also help us keep track of where someone's base is located. Always keep track of where your claims are located to make solving issues easier. These coordinates may be required in tickets.

Reviewer 2: ```suggestion

The coordinates get updated and posted into the [#ds9-coordinates](https://discord.com/channels/297301054930944011/965230296226222150) channel. These coordinates also help us keep track of where someone's base is located. Always keep track of where your claims are located to make solving issues easier. These coordinates may be required in tickets.
```

- `/trigger nc_inOverworld`: Converts current coordinates to nether coordinates.Can be used as a way to efficiently calculate coordinates for portals.
- `/aboutdc`: Shows information about Derpitcraft.
- `/tm | /tmsg | /teammsg`: Message within your team.

### Using proximity chat (Simple Voice Chat)

Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. You can download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. This system can only work with Java clients. We recommend Bedrock clients use Discord.

Once you have downloaded the mod, it should work out of the box. Read more about the mod on it's wiki (https://modrepo.de/minecraft/voicechat/overview). There are very detailed explanations available. If you still do not understand how to use this mod, or it doesn't work, raise a ticket.

You can also join voice chat "groups". This is an easy way to voice chat between a group of players. This is recommended for teams. You can password protect these groups from the interactive creation wizard.

> ### Coordinates and Claims

Reviewer 2: This section is specifically about the DCCoordinates command and not Coordinates in general. This is ambiguous.

Author: made it explicit
